src/style.css

```diff
@@ -0,0 +1,117 @@
+body {
+  margin: 0;
+}
+
+.container {
+  width: 200px;
+  padding-top: 29px;
+  padding-bottom: 15px;
+  border: 1px solid #f3f3f3;
+  box-sizing: border-box;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.card-image {
+  width: 160px;
+  height: 134px;
+}
+
+.card-text {
```

Comment:
.card-text is BEM-block now, so it's not supposed to have margins. But better make it BEM-element instead